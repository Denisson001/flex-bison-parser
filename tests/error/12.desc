Runtime error - division by zero