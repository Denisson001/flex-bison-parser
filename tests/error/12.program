{
    a = 0;
    b = 4 / a;
    c = 5 % a;
}