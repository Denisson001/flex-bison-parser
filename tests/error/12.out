Interpreter: runtime error - Division by zero
